### ISSUE TITLE
System won't boot when lacking of free space and trying to opting in steam beta

### ISSUE THREAD

**Issue by slagerfin** [ValveSoftware/SteamOS#310]
One of my games was not working so I thought to give the beta a shot. I opted the beta mode in system menu. Unfortunately I didn't realize that I had lack of free space and now I can't boot to Steam anymore. It just gives me error that says I should have at least 250MB of free space to install beta mode.

**Comment by ProfessorKaos64** [comment]
Can't you enter grub-recovery from the bootloader and remove the offending extra files to free up space? Is it complaining about /home/steam being full? or what directory? You could also plug your HDD into an adapter and load a live ISO of Debian to mount the drive and check things out.

**Comment by Vash63** [comment]
I'm assuming that error message is coming from Steam itself, with it up you should try hitting ctrl+alt+f2 to go to a terminal, and try to clear up space manually.

This would probably be a good feature request, it should definitely give a warning any time you're getting close to running out of space.

**Comment by sharkwouter** [comment]
This is the same issue as: https://github.com/ValveSoftware/SteamOS/issues/352

It's a pretty serious problem, as there is no easy way to recover from this.

**Comment by Vash63** [comment]
I agree that it's a serious problem, but that's no reason to be creating duplicates for the same bug, especially intentionally.

**Comment by sharkwouter** [comment]
I didn't know it was a duplicate before, should I close it?

**Comment by Vash63** [comment]
Valve doesn't seem to police this github tracker very much, but if two issues are the same it's generally best to only have one so it gets more visibility and is easier to read.
